website/source/docs/enterprise/runs/variables-and-configuration.html.md

```diff
@@ -75,9 +80,42 @@ For any of the `GITHUB_` attributes, the value of the environment variable will
 be the empty string (`""`) if the resource is not connected to GitHub or if the
 resource was created outside of GitHub (like using `terraform push`).
 
+### Personal Variables
+
+Personal variables can be created at the Environment level. All Personal
+Environment variables are private and scoped to the user that created them. Just
+like Environment variables, they are injected into the virtual environment
+during the `plan` and `apply` phases.
+
+Personal Environment variables can be used to override Environment variables on
+a per-user basis. You can add, edit, and delete environment variables from the
```

Comment:
also what are your thoughts on including a screenshot in that top section like the below?
https://www.dropbox.com/s/onng6647c77xgoi/Screenshot%202017-05-03%2010.13.35.png?dl=0

Useful or unnecessary?